swap/swap.go

```diff
@@ -173,7 +173,11 @@ func (s *Swap) Add(amount int64, peer *protocols.Peer) (err error) {
 	// that the balance is *below* the threshold
 	if newBalance <= -s.paymentThreshold {
 		log.Warn("balance for peer went over the payment threshold, sending cheque", "peer", peer.ID().String(), "payment threshold", s.paymentThreshold)
-		return s.sendCheque(peer.ID())
+		swapPeer, ok := s.getPeer(peer.ID())
+		if !ok {
+			return fmt.Errorf("error while getting peer: %s", peer)
```

Comment:
This should actually be "no peer found", as we don't print any error in the text

Reply:
you mean something like `return fmt.Errorf("peer %s not found", peer)`?

Reply:
yes

Reply:
fixed, thanks